commit 6d690dcac92a84f98fd774862628ff871b713660
Author: Akinobu Mita <akinobu.mita@gmail.com>
Date:   Sat May 12 10:36:53 2007 -0700

    fault injection: disable stacktrace filter for x86-64
    
    Disable stacktrace filter support for x86-64 for now.  Will be enable when we
    can get the dwarf2 unwinder back.
    
    Signed-off-by: Akinobu Mita <akinobu.mita@gmail.com>
    Acked-by: Andi Kleen <ak@suse.de>
    Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
    Signed-off-by: Linus Torvalds <torvalds@linux-foundation.org>

diff --git a/lib/Kconfig.debug b/lib/Kconfig.debug
index ee05b8a061b5..fbc5c622acb0 100644
--- a/lib/Kconfig.debug
+++ b/lib/Kconfig.debug
@@ -438,6 +438,7 @@ config FAULT_INJECTION_DEBUG_FS
 config FAULT_INJECTION_STACKTRACE_FILTER
 	bool "stacktrace filter for fault-injection capabilities"
 	depends on FAULT_INJECTION_DEBUG_FS && STACKTRACE_SUPPORT
+	depends on !X86_64
 	select STACKTRACE
 	select FRAME_POINTER
 	help